### ISSUE TITLE
Document authentication with R client

### ISSUE THREAD

**Issue by m-mohr** [openEOPlatform/documentation#6]

**Comment by przell** [comment]
@m-mohr: What's the deadline?

**Comment by m-mohr** [comment]
Not sure myself, is it Phiweek? @pstranner

**Comment by m-mohr** [comment]
This is the page we are talking about: https://docs.openeo.cloud/getting-started/r

**Comment by przell** [comment]
Also related to this: [https://github.com/Open-EO/openeo-r-client/issues/69](https://github.com/Open-EO/openeo-r-client/issues/69)
I also wouldn't consider it a hard requirement to be there at a certain timing. But for sure we'll try. And we'll need it anyway later on for the R client project addition.

**Comment by m-mohr** [comment]
Good point, we may not make the R client work with OIDC (at least without the user needing to request a client ID/secret and I'm not sure that's what EGI let's us do). In this case, we may want to remove the R getting started guide for now.

**Comment by przell** [comment]
We have the kick-off for the r-client project on 09.09.2021. We could focus on documentation and authentication first to make the getting started guide available again.

**Comment by m-mohr** [comment]
Once we make this available, we also need to change the link at https://openeo.cloud/#clients

**Comment by vardizzo-lab** [comment]
@przell : please could you send an email to checkin-support@mailman.egi.eu and request technical support for this client's configuration to EGI Check-in Support Unit?
Thanks.

**Comment by m-mohr** [comment]
@vardizzo-lab We have pending client IDs (`openeo-platform` and `openeo-platform-r`) that simply need to get accepted and deployed, which I hope will happen soon. I don't think we need to open a support ticket right now.

**Comment by vardizzo-lab** [comment]
I have checked and it seems that some information is missing in both configurations. Enter it to allow approval.

**Comment by m-mohr** [comment]
@vardizzo-lab I've already entered all requested information this morning. Please clarify what is missing?
Please note that I can't remove the localhost URLs as requested as we use them in production!

**Comment by vardizzo-lab** [comment]
@m-mohr please add the support contact in the configuration client ( Contacts section), so the team can proceed with the review.
